src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const stateObject = state;
```

Comment:
I think variable is not required for this task. You can simply use 'state' as a object to modify

Reply:
it's a work of habit :) removed.